### PR TITLE
Add operator.openshift.io/v1/network to relatedObjects for must-gather

### PR DESCRIPTION
**Pull request by rcarrillocruz**
Fixes [SDN-1344](https://issues.redhat.com/browse/SDN-1344)

### CODE REVIEW

**Comment by rcarrillocruz** [comment]
With this running oc get clusteroperators network -oyaml shows:

...
  - group: operator.openshift.io
    name: cluster
    resource: network
  versions:
  - name: operator
    version: 4.7.0-0.ci-2020-11-11-075910

File: pkg/controller/operconfig/operconfig_controller.go [hunk 1]
```diff
@@ -241,6 +241,13 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		Name:     names.APPLIED_NAMESPACE,
 	})
 
+	// Add operator.openshift.io/v1/network to relatedObjects for must-gather
+	relatedObjects = append(relatedObjects, configv1.ObjectReference{
+		Group:    "operator.openshift.io",
+		Resource: "network",
```
Comment (tremes) [inline pkg/controller/operconfig/operconfig_controller.go:247]: Oh. I just realized that the resource name is actually "networks" and not "network". @rcarrillocruz Can you please fix it?

Comment (tremes) [reply pkg/controller/operconfig/operconfig_controller.go:247]: I created https://github.com/openshift/cluster-network-operator/pull/877

**Pull request merged**